packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -149,6 +149,7 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
   Future<void> animateCamera(
     CameraUpdate cameraUpdate, {
     required int mapId,
+    Duration? animationDuration,
```

Comment:
Please add documentation for what does this parameter do, what means to be null etc.

Reply:
@cyanglaz Should I add the doc on the parameter or on the function?

Reply:
On the function. It could be something like:
/// The `animationDuration` parameter indicates how long it takes for the animation to complete. If `animationDuration` is `null`, a default animation duration will be provided.